docs/documentation/pages/DECKHOUSE-FAQ_RU.md

```diff
@@ -965,6 +965,228 @@ kubectl -n d8-system exec -ti svc/deckhouse-leader -c deckhouse -- deckhouse-con
 
    Вывод команды должен быть пуст.
 
+### Как переключить Deckhouse EE на CSE?
+
+{% alert %}
+Инструкция подразумевает использование публичного адреса container registry: `registry-cse.deckhouse.ru`. В случае использования другого адреса container registry измените команды или воспользуйтесь [инструкцией по переключению Deckhouse на использование стороннего registry](#как-переключить-работающий-кластер-deckhouse-на-использование-стороннего-registry).
+{% endalert %}
+
+{% alert %}
+В Deckhouse CSE не поддерживается работа облачных кластеров и ряда модулей.
+{% endalert %}
+
+{% alert %}
+На текущий момент мигрировать на CSE редакцию можно только с релиза 1.58.
+{% endalert %}
+
+Для переключения кластера Deckhouse Enterprise Edition на Certified Security Edition выполните следующие действия:
+Все команды выполняются на мастер ноде действующего EE кластера с версией релиза 1.58.
+
+1. Убедитесь, что используемые в кластере модули поддерживаются в CSE редакции.  
+Например на текущий момент в CSE редакции отсутствует модуль CertManager, поэтому перед его отключением необходимо перевести `Режим работы HTTPS` для связанных компонентов (например [user-authn](https://deckhouse.ru/products/kubernetes-platform/documentation/v1.58/modules/150-user-authn/configuration.html#parameters-https-mode) или [prometheus](https://deckhouse.ru/products/kubernetes-platform/documentation/v1.58/modules/300-prometheus/configuration.html#parameters-https-mode
+) ) на альтернативаные варианты.
+
+    Отключить не поддерживаемые модули в CSE редакции можно следующими командами:
+
+   ```shell
+   kubectl -n d8-system exec -it deploy/deckhouse bash 
+
+   kubectl get modules.deckhouse.io | grep Enabled | grep -v -E "(admission-policy-engine|cni-flannel|common|control-plane-manager|chrony|deckhouse|ingress-nginx|kube-dns|kube-proxy|log-shipper|local-path-provisioner|loki|monitoring-kubernetes|node-manager|operator-prometheus|registrypackages|runtime-audit-engine|operator-trivy|prometheus|user-authn|user-authz|priority-class|cloud-data-crd|operator-prometheus-crd|prometheus-crd|user-authn-crd|vertical-pod-autoscaler-crd|flow-schema|prometheus-metrics-adapter|monitoring-deckhouse|monitoring-kubernetes-control-plane|gost-integrity-controller|snapshot-controller-crd)" | awk {'print "deckhouse-controller module disable ", $1'} | bash && exit
```

Comment:
- Firstly we need to get unsupported modules list to solve problems before release change.

Reply:
done, I deleted hardcode with list of unsupported modules and add point about check modules which will be disabled:
https://github.com/deckhouse/deckhouse/blob/aaaa00124487f365b106980ab5257fa5d19d1088/docs/documentation/pages/DECKHOUSE-FAQ_RU.md?plain=1#L1076-L1090